File: CodeForces/Java/CF_155A_I_love_%username%.java
//  Problem: 155A. I_love_%username%
//  Link: https://codeforces.com/problemset/problem/155/A

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner scan = new Scanner(System.in);
    int n = scan.nextInt();
    
    int first = scan.nextInt();
    int min = first;
    int max = first;
    int amazing = 0;
    
    for (int i = 1; i < n; i++) {
        int score = scan.nextInt();
        if (score > max) {
            max = score;
            amazing++;
        } else if (score < min) {
            min = score;
            amazing++;
        }
    }
    
    System.out.println(amazing);
    scan.close();
  }
}

File: CodeForces/Python/CF_155A_I_love_%username%.py
# Problem: 155A. I_love_%username%
# Link: https://codeforces.com/problemset/problem/155/A

n = int(input())
scores = list(map(int, input().split()))

min_score = scores[0]
max_score = scores[0]
amazing = 0

for i in range(1, n):
    if scores[i] > max_score:
        max_score = scores[i]
        amazing += 1
    elif scores[i] < min_score:
        min_score = scores[i]
        amazing += 1

print(amazing)
